src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.Simple.cs

```diff
@@ -523,6 +523,7 @@ public void OperationsOnUnconnectedClient()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #16934")] //Hangs forever in desktop as it doesn't have cancellation support
```

Comment:
I wish you could do this, which is preferable as it keeps all the issues in ActiveIssue, where we can grep them.

`[ActiveIssue("https://github.com/dotnet/coreclr/issues/16934", TestPlatforms.AnyUnix)]`

However it looks like we are missing that infra.